src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+  margin-bottom: 24px;
+
+  &__star {
+    background: url(../images/star.svg) no-repeat center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+}
+
+.stars--4 .stars__star:nth-child(4n),
+.stars--4 .stars__star:nth-child(-n+4) {
+  background: url(../images/star-active.svg) no-repeat center;
+}
```

Comment:
shold be styles for all possible amount of active stars
and pay attention on this point from checklist
![Screenshot from 2023-04-17 21-16-18](https://user-images.githubusercontent.com/33029864/232574765-7273fccc-0bd1-46c8-84c8-76c532526ad5.png)